src/Microsoft.ML.Transforms/OneHotEncoding.cs

```diff
@@ -22,7 +22,24 @@
 
 namespace Microsoft.ML.Transforms
 {
-    /// <include file='doc.xml' path='doc/members/member[@name="CategoricalOneHotVectorizer"]/*' />
+    /// <summary>
+    /// <see cref="ITransformer"/> resulting from fitting a <see cref="OneHotEncodingEstimator"/>.
+    /// </summary>
+    /// <remarks>
+    /// <para>
+    /// The OneHotEncodingTransformer passes through a data set, operating on text columns, to build a dictionary of categories.
+    /// For each row, the entire text string appearing in the input column is defined as a category.</para>
+    /// <para>The output of this transform is an indicator vector.
```

Comment:
Isn't the output dependent on the output kind?

Reply:
I'll look into it and update this in another commit.